wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -225,6 +225,85 @@ executable cardano-node
                       UndecidableInstances
                       MonadFailDesugaring
 
+executable cardano-integration-test
```

Comment:
Because it requires the full wallet node to be running, and currently we don't have any easy way to run it from the code. Nothing stops us from changing this in the future, and this is indeed what I had in mind myself.
The current issue is the lack of `cardano-runner` or similar which is required for `dbgen` as well, not to mention that it would be very beneficial to replace the bash scripts with an actual configurable and flexible Haskell implementation.

Reply:
Sounds great :smile:

Reply:
@ksaric Can you please log this as a ticket, so we don't forget? (i.e. turning this into a `test-suite` so CI can run it). Not entirely sure if we have at this stage a parent user story to reference this for, but better logged as an orphan task that not logged at all, imho 😉

Reply:
Here we go - https://iohk.myjetbrains.com/youtrack/issue/CSL-2401